components/scream/src/control/tests/ad_tests.yaml

```diff
@@ -9,8 +9,6 @@ Initial Conditions:
     A: 1.0
     B: 2.0
     C: 3.0
-    D: [4.0, 5.0]
```

Comment:
By removing this are we no longer testing if it is possible to pass a vector quantity to initialize a vector field?

Reply:
Added back testing for that feature.